templates/layouts/navbar.html.ep

```diff
@@ -40,7 +40,7 @@
                   %= link_to 'Audit log' => url_for('audit_log') => class => 'dropdown-item'
                   % }
                   %= tag 'div' => class => 'dropdown-divider'
-                  %= link_to 'Logout' => url_for('logout') => class => 'dropdown-item'
+                  %= link_to 'Logout' => url_for('/logout') => 'data-method' => 'post' => class => 'dropdown-item'
```

Comment:
Let's remove `$r->get('/logout')->to('session#destroy');` from WebAPI.pm as well - `get` methods shouldn't modify state of session. Then we will also need to tweak two tests which use `$mc/curl /logout`

Reply:
Yep, thanks for pointing it out